hack/allocate.sh

```diff
@@ -298,6 +301,7 @@ EOF
 
 dapr_runtime() {
```

Comment:
Has the [dashboard](https://github.com/knative/func/pull/1819/files?diff=split&w=1#diff-3900b46a5058b65369632a94a58bad6cda0e7e9e8bf991c30e20a32d13a0a527L288) image changed between versions?
If so the `fixup_dapr_dashboard_deployment()` function should be updated too.

Reply:
Now I am trying installing dapr on k8s and it is not setting "dashboard" at all, weird.

Reply:
They probably removed dashboard from default installation via helm hart maybe? :man_shrugging:

Reply:
OK, thanks for checking.